final-project/src/tier1/envs/aerial_robot_finalproject_tier1_task.py

```diff
@@ -402,12 +435,14 @@ def compute_quadcopter_reward(root_positions, root_quats, root_linvels, root_ang
 
     # resets due to episode length
     reset = torch.where(progress_buf >= max_episode_length - 1, ones, die)
-    reset = torch.where(torch.norm(root_positions, dim=1) > 20.0, ones, reset) # out of bounds for a norm distance of 20.0
-
+    print("reset1",reset)
+    reset = torch.where(torch.norm(root_positions, dim=1) > 35.0, ones, reset) # out of bounds for a norm distance of 20.0
+    print("reset2",reset)
     # Above a certain self.counter number, if the z coordinate is too close to ground, then reset
-    if counter > 500:
-        ground_threshold = 0.25
+    if counter > -1:
```

Comment:
The problem we with this is that sometimes when the drone starts too low the ground, the episode immediately terminates.

Also, sorry but where is ground_threshold initialized? I know it's somewhere but I can't find it for some reason

Reply:
I am just setting ground threshold everytime to ground_threshold = 0.15.  The lowest it sets at initiation is or should be .20.  
Issue is that sometimes it goes below or negative and sometimes it clamps down to .2 - need to track down in Aerial Gym or Isaac.